datadog/__init__.py

```diff
@@ -39,7 +39,7 @@ def initialize(
     statsd_port=None,  # type: Optional[int]
     statsd_disable_aggregator=True,  # type: bool
```

Comment:
Similar to my other comment, but we should be consistent. maybe rename this to `statsd_disable_aggregation`?

Reply:
Yeah this sounds good to me, I just changed the name to one that someone suggested